### PR TITLE
DOCSP-35064 Add Sybase Mentions to Corpus

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Add Sybase mentions to the connection string page and other supplementary pages where the other RDMS's are mentioned. 

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-35064/connection-strings/relational-database-connection-strings/#sybase

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-35064/jobs/creating-jobs/

## JIRA

https://jira.mongodb.org/browse/DOCSP-35064

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66280179b67dbf804a902a0f

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
@tomhollander This PR includes Sybase mentions on various pages and is ready for a look. I checked with Satya in slack and he confirmed DDL files are not supported via Sybase.

**Review by tomhollander — APPROVED** [review]

File: source/projects/create-project-loading-schema-files.txt [hunk 1]
```diff
@@ -103,6 +103,11 @@ to generate DDL files from your relational system.
 
       #. Select :guilabel:`Options` and enable :guilabel:`Include CREATE DATABASE statement`.
 
+   .. tab:: Sybase
+      :tabid: sybase-ddl
+
+      Importing DDL files with SybaseASE is not supported.
```
Comment (sarah-olson-mongodb) [inline source/projects/create-project-loading-schema-files.txt:109]:
```suggestion
      Importing DDL files with Sybase ASE is not supported.
```

**Review by sarah-olson-mongodb — COMMENTED** [review]
One spacing issue to fix up before we merge. 
Also, it occurs to me that it might be worth standardizing on either "Sybase" or "Sybase ASE". Right now we're using both, which isn't wrong of course, but may be worth having a quick look at while you're in there.

File: source/projects/create-project-loading-schema-files.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/projects/create-project-loading-schema-files.txt]: Sure @sarah-olson-mongodb, I have updated to `Sybase ASE` throughout for consistency.

**Review by sarah-olson-mongodb — APPROVED** [review]

**Pull request merged**